flink-runtime/src/main/java/org/apache/flink/runtime/source/event/AddSplitEvent.java

```diff
@@ -30,15 +33,23 @@ Licensed to the Apache Software Foundation (ASF) under one
 public class AddSplitEvent<SplitT> implements OperatorEvent {
 
 	private static final long serialVersionUID = 1L;
-
-	private final List<SplitT> splits;
-
-	public AddSplitEvent(List<SplitT> splits) {
-		this.splits = splits;
+	private final int serializerVersion;
+	private final List<byte[]> splits;
```

Comment:
Nit: For serializable classes it is nice to put the concrete type of the list there, because not all list implementations are serializable. And because this is a purely internal field, typing this to `ArrayList` wouldn't change any abstractions.